packages/components/src/entities/SampleTimelinePageBase.tsx

```diff
@@ -154,7 +156,12 @@ export const SampleTimelinePageBaseImpl: FC<OwnProps & InjectedQueryModels> = me
         if (value && value.startsWith('materialinputs/')) {
             return getMaterialDataInputDisplay(SAMPLES_KEY, field.toLowerCase(), value, 'materialinputs/');
         } else if (value && value.startsWith('datainputs/')) {
-            return getMaterialDataInputDisplay(SOURCES_KEY, field.toLowerCase(), value, 'datainputs/');
+            return getMaterialDataInputDisplay(
+                getPrimaryAppProperties().dataclassUrlPart,
```

Comment:
This parameter could probably be removed in favor of making this call in the `getMaterialDataInputDisplay` method itself, but no big deal.